### PR TITLE
Hercules MP3e2: New mapping

### PR DESCRIPTION
**Pull request by pyxl-dev**
Here's a new mapping for Hercules DJ Control MP3 e2 controller. It's intended to respect the original mapping of the controller.

### CODE REVIEW

File: res/controllers/Hercules DJ Control MP3 e2.bulk.xml [hunk 1]
```diff
@@ -1,13 +1,12 @@
 <?xml version='1.0' encoding='utf-8'?>
-<MixxxControllerPreset mixxxVersion="1.11+" schemaVersion="1">
+<MixxxControllerPreset mixxxVersion="2.0+" schemaVersion="1">
     <info>
-        <name>Hercules DJ Control MP3 e2 / MP3 LE / Glow</name>
-        <author>Neale Pickett</author>
-        <description>Hercules DJ Control MP3 e2 / MP3 LE / Glow mapping. These three controllers have identical controls and share the same mapping.</description>
+        <name>Hercules DJ Control MP3 e2</name>
```
Comment (Holzhaus) [inline res/controllers/Hercules DJ Control MP3 e2.bulk.xml:4]: I had the impression that these are basically the same controller. Any reason you removed it?

**Review by Holzhaus — COMMENTED** [review]
Corresponding documentation PR: https://github.com/mixxxdj/manual/pull/387

File: res/controllers/Hercules DJ Control MP3 e2.bulk.xml (hunk 1 above)
Comment (0xYoan) [reply res/controllers/Hercules DJ Control MP3 e2.bulk.xml:4]: Yes you're right, I wasn't sure about this then I removed it. I'll add it again.

File: .gitignore [hunk 2]
```diff
@@ -36,3 +36,4 @@ CMakeSettings.json
 
 # VisualStudio project files
 /.vs/
+.vscode
```
Comment (uklotzde) [inline .gitignore:39]: Please revert this change, see #3861.

**Review by uklotzde — DISMISSED** [review]

**Review by Holzhaus — COMMENTED** [review]
Thank you @0xYoan.

@uklotzde I have no way to test this. Shall we merge anyway? This has been long open already. The manual PR targets the 2.3 branch, so I guess this one should too?

**Comment by Holzhaus** [comment]
Rebased and squashed.

**Comment by uklotzde** [comment]
@Holzhaus Manual PR has been approved. Merge?

File: res/controllers/Hercules DJ Control MP3 e2.bulk.xml [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
+<MixxxControllerPreset mixxxVersion="2.0+" schemaVersion="1">
     <info>
         <name>Hercules DJ Control MP3 e2 / MP3 LE / Glow</name>
-        <author>Neale Pickett</author>
+        <author>Yoan Ancelly</author>
         <description>Hercules DJ Control MP3 e2 / MP3 LE / Glow mapping. These three controllers have identical controls and share the same mapping.</description>
         <manual>hercules_djcontrol_mp3_e2</manual>
         <devices>
             <product protocol="bulk" vendor_id="0x06f8" product_id="0x0b105" in_epaddr="0x82" out_epaddr="0x03" />
-            <product protocol="bulk" vendor_id="0x06f8" product_id="0x0b120" in_epaddr="0x82" out_epaddr="0x03" />
```
Comment (Holzhaus) [inline res/controllers/Hercules DJ Control MP3 e2.bulk.xml:10]: Why was this removed? Will it still work with the other controllers?

Comment (uklotzde) [reply res/controllers/Hercules DJ Control MP3 e2.bulk.xml:10]: @0xYoan Ping

**Review by uklotzde — CHANGES_REQUESTED** [review]
Please clarify why product_id=0x0b120 has been removed. The comment still states that the mapping works for all listed devices!

File: res/controllers/Hercules DJ Control MP3 e2.bulk.xml (hunk 3 above)
Comment (reinfire) [reply res/controllers/Hercules DJ Control MP3 e2.bulk.xml:10]: Please dont remove this line, it is needed for the Hercules Glow (Green Version).

Here the dmesg from the Hercules Glow Green-Version:

[10496.108493] usb 2-2.1: new full-speed USB device number 8 using ehci-pci
[10496.198155] usb 2-2.1: New USB device found, idVendor=06f8, idProduct=b120, bcdDevice= 1.11
[10496.198165] usb 2-2.1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[10496.198171] usb 2-2.1: Product: DJControl MP3 LE
[10496.198174] usb 2-2.1: Manufacturer: Hercules
